goat_launch/params/locomotor_params.yaml

```diff
@@ -1,31 +1,30 @@
 locomotor:
```

Comment:
pretty sure this needs to be removed and the rest of things outdented. pretty sure